File: 0804/app/src/main/java/com/example/a0804/ThirdActivity.java
package com.example.a0804;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;

public class ThirdActivity extends AppCompatActivity {

    private static final String TAG = "ThirdActivity";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_third);

        Log.d(TAG, "ThirdActivity запущена");
    }

    @Override
    protected void onStart() {
        super.onStart();
        Log.d(TAG, "ThirdActivity: onStart()");
    }

    @Override
    protected void onResume() {
        super.onResume();
        Log.d(TAG, "ThirdActivity: onResume()");
    }

    @Override
    protected void onPause() {
        super.onPause();
        Log.d(TAG, "ThirdActivity: onPause()");
    }

    @Override
    protected void onStop() {
        super.onStop();
        Log.d(TAG, "ThirdActivity: onStop()");
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
        Log.d(TAG, "ThirdActivity: onDestroy()");
    }
}

File: 09_06/app/src/main/java/com/example/a09_06/ThirdActivity.kt
package com.example.a09_06

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class ThirdActivity: AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_third)
        val text = intent.getStringExtra("text") ?: ""
        findViewById<TextView>(R.id.tvResult).text = text
    }
}
